### PR TITLE
Prevent author and reviewer names from wrapping in chapter pages

### LINKED ISSUE #1471: Prevent names from wrapping in chapter bylines
image

Ian's first and last name should always be on the same line. Prevent the name from wrapping by setting `white-space: nowrap;` on a new `.byline .author` style in [page.css](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/src/static/css/page.css).

### PR DESCRIPTION
**Pull request by sudheendrachari**
Fixes #1471 by adding _white-space: nowrap_ for
author and reviewer names
Author text wrap

### CODE REVIEW

File: src/static/css/page.css [hunk 1]
```diff
@@ -196,6 +196,11 @@
   font-weight: normal;
 }
 
+.byline .author,
+.byline .reviewer {
+  white-space: nowrap;
+}
+
```
Comment (bazzadp) [inline src/static/css/page.css:203]: We should include analysts and reviewers in this too.

Should we just change this to the generic `.byline a`?:

```suggestion
.byline a {
  white-space: nowrap;
}

```

**Comment by bazzadp** [comment]
Love the video btw! 😁

Can you also add yourself to 2020.json in the `developers` team as part of this? Just noticed you're not in there so not getting credit on [our Contributors page](https://almanac.httparchive.org/en/2020/contributors?teams=developers).

**Comment by bazzadp** [comment]
Can you merge `main` into this branch to pick up #1535 to avoid check errors that will happen next time you change this branch.

**Review by rviscomi — APPROVED** [review]
LGTM once @bazzadp's feedback is resolved. Thanks for working on this @sudheendrachari!

**Review by bazzadp — APPROVED** [review]

**Pull request merged**